src/callback.js

```diff
@@ -3,18 +3,25 @@ import { useCallback, useEffect, useRef } from 'react';
 export default function useDebouncedCallback(callback, delay, deps) {
   const functionTimeoutHandler = useRef(null);
   const debouncedFunction = useCallback(callback, deps);
+  const cancel = useCallback(() => {
+    clearTimeout(functionTimeoutHandler.current);
+  }, [functionTimeoutHandler.current]);
 
   useEffect(
     () => () => {
-      clearTimeout(functionTimeoutHandler.current);
+      cancel();
     },
     []
   );
 
-  return (...args) => {
+  const debouncedCallback = (...args) => {
     clearTimeout(functionTimeoutHandler.current);
     functionTimeoutHandler.current = setTimeout(() => {
       debouncedFunction(...args);
     }, delay);
   };
+
+  debouncedCallback.cancel = cancel;
```

Comment:
Besides `useDebouncedCallback` the project has `useDebounce` hook. it's better to keep a similar contract for both cases. 
It could be `[value, cancelValue]` for `useCallback` hook and `[callback, cancelCallback]` for `useDebouncedCallback`

Also, it's better to keep identical features for both hooks.

Reply:
Thank you @xnimorz 

I agree with you. I will fixed tonight